app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager.rb

```diff
@@ -31,6 +31,12 @@ class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager < ManageI
            :autosave    => true,
            :dependent   => :destroy
 
```

Comment:
@Kuldip-Nanda you have to add `require_nested :PlacementGroup` here so that the STI type is detected properly

Reply:
@miq-bot cross-repo-tests https://github.com/ManageIQ/manageiq-schema/pull/644, https://github.com/ManageIQ/manageiq/pull/21807